src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/HearingGetConsumerTest.java

```diff
@@ -299,7 +284,18 @@ void shouldFailGetHearingWithNotFound() {
                 SERVICE_AUTHORIZATION_TOKEN,
                 NOT_FOUND_CASE_ID,
                 null
-            )).extracting("status").isEqualTo(NOT_FOUND);
+            )).extracting(STATUS).isEqualTo(NOT_FOUND);
 
     }
+
+    private HearingGetResponse getExpectedHearingGetResponse() throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.configure(WRITE_DATES_AS_TIMESTAMPS, false);
+        objectMapper.configure(READ_UNKNOWN_ENUM_VALUES_AS_NULL, true);
```

Comment:
These are already configured in the application.yaml properties

Reply:
True, I've removed these lines